tensorboard/plugins/profile/trace_events_json.py

```diff
@@ -90,7 +90,7 @@ def _event(self, event):
       result['ph'] = _TYPE_INSTANT
       result['s'] = _SCOPE_THREAD
     for key in dict(event.args):
-      if result['args'] is None:
+      if 'arg' not in result:
```

Comment:
Not sure how the code passes the test, I think there's an obvious typo here:
'arg' => 'args'

Reply:
Thanks for catching another bug.... Updated the test to cover this as well.